manual/configuration.md

```diff
@@ -565,6 +565,14 @@ You can also disable *all* cops with
 # rubocop:enable all
 ```
 
+You can also disable cops using the todo alias.
```

Comment:
I'd mention what this is an alias of exactly and what's the value of having it. Currently this section is a bit confusing.

Reply:
Updated the text